projects/1.16/assets/xaeros-minimap/xaerobetterpvp/lang/zh_cn.json

```
@@ -1,354 +1,457 @@
+  "gui.xaero_not_ingame": "你必须处于游戏中才能使用编辑模式！",
+  "gui.xaero_welcome_edit_mode": "欢迎进入编辑模式！",
+  "gui.xaero_howto_select": "左键选择一个界面。",
+  "gui.xaero_howto_drag": "左键拖动来移动界面。",
+  "gui.xaero_howto_deselect": "右键来取消选择一个界面。",
+  "gui.xaero_howto_center": "按下“C”来居中或取消居中一个界面。",
+  "gui.xaero_different_centered": "部分界面有着不同的居中样式。",
+  "gui.xaero_howto_flip": "按下“F”来翻转或取消翻转一个界面。",
+  "gui.xaero_howto_settings": "按下“S”来快速访问所选界面的设置。",
+  "gui.xaero_howto_preset": "点击“选择一个预设”按钮来选择一个预设。",
+  "gui.xaero_howto_save": "点击“确认”按钮来保存你的更改。",
+  "gui.xaero_howto_cancel": "点击“取消”按钮来取消你的更改。",
+  "gui.xaero_minimap_settings": "小地图设置",
+  "gui.xaero_miscellaneous_settings": "杂项设置",
+  "gui.xaero_notification_settings": "通知设置",
+  "gui.xaero_better_pvp_settings": "Better PvP 设置",
+  "gui.xaero_server_disabled": "服务器可能禁用了此 mod 的一些功能。",
+  "gui.xaero_waypoints": "路径点",
+  "gui.xaero_box_potion_effects": "一个显示当前激活的药水效果的界面。",
+  "gui.xaero_box_armour_status": "一个显示当前装备和主手物品信息的界面。",
+  "gui.xaero_box_sprint": "使用一个单独的按键（可配置的）来冲刺。",
+  "gui.xaero_box_sneak": "允许你切换潜行状态，并且在潜行状态中不需要按住任何按键。",
+  "gui.xaero_box_notifications": "为一些信息启用屏幕通知，如生命值低、饱食度低、氧气量低、苦力怕预警、或者骷髅怪预警。",
+  "gui.xaero_box_xp": "获得或者失去经验值时将会有提示。",
+  "gui.xaero_box_quick_use": "一个类似于动作条的热键系统，可以让你在不将主手物品换掉的情况下使用指定的物品。按住热键来持续使用物品，就像按住右键时一样。比如：放置火把、TNT、扔药水、吃东西、喝东西等等。",
+  "gui.xaero_box_minimap": "一个显示你周围环境与实体的小地图。",
+  "gui.xaero_centered": "已居中:",
+  "gui.xaero_press_c": "(按下 C)",
+  "gui.xaero_flipped": "已翻转:",
+  "gui.xaero_press_f": "(按下 F)",
+  "gui.xaero_preset_fancy": "幻想",
+  "gui.xaero_preset_classic": "经典",
+  "gui.xaero_preset_social": "社交",
+  "gui.xaero_potion_status": "药水效果",
+  "gui.xaero_archery_status": "显示弓箭剩余数量",
+  "gui.xaero_sprint": "更好的冲刺",
+  "gui.xaero_sneak": "切换潜行",
+  "gui.xaero_enchants_color": "附魔颜色",
+  "gui.xaero_durability": "耐久度",
+  "gui.xaero_hunger_setting": "饱食度低提醒",
+  "gui.xaero_hp_setting": "健康值低提醒",
+  "gui.xaero_explosion_setting": "苦力怕预警",
+  "gui.xaero_being_shot_setting": "骷髅怪预警",
+  "gui.xaero_xp_setting": "经验掉落",
+  "gui.xaero_custom_settings": "自定义设置",
+  "gui.xaero_reset_defaults": "恢复默认设置",
+  "gui.xaero_quick_use": "快速使用物品",
+  "gui.xaero_show_enchants": "显示附魔",
+  "gui.xaero_cave_maps": "洞穴模式",
+  "gui.xaero_display_teams": "显示其他队伍",
+  "gui.xaero_display_waypoints": "显示地图上路径点",
+  "gui.xaero_display_players": "显示玩家",
+  "gui.xaero_display_mobs": "显示怪物",
+  "gui.xaero_display_items": "显示物品",
+  "gui.xaero_display_other": "显示其他",
+  "gui.xaero_zoom": "小地图缩放",
+  "gui.xaero_minimap_size": "小地图尺寸",
+  "gui.xaero_entity_amount": "实体数量限制",
+  "gui.xaero_black": "黑色",
+  "gui.xaero_dark_blue": "深蓝色",
+  "gui.xaero_dark_green": "墨绿色",
+  "gui.xaero_dark_aqua": "深青色",
+  "gui.xaero_dark_red": "暗红色",
+  "gui.xaero_dark_purple": "深紫色",
+  "gui.xaero_gold": "金色",
+  "gui.xaero_gray": "灰色",
+  "gui.xaero_dark_gray": "深灰色",
+  "gui.xaero_blue": "蓝色",
+  "gui.xaero_green": "绿色",
+  "gui.xaero_aqua": "青色",
+  "gui.xaero_red": "红色",
+  "gui.xaero_purple": "亮紫色",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "gui.xaero_purple": "浅紫色",
```